apps/site/assets/ts/stop/components/Departures.tsx

```diff
@@ -64,20 +84,47 @@ const Departures = ({
             modeButtonsToShow={availableModes(routes)}
           />
         )}
-
-        {filteredRoutes.map(routeWithDirections => (
-          <RouteCard
-            key={routeWithDirections.route.id}
-            route={routeWithDirections.route}
-            directions={routeWithDirections.directions}
-            stop={stop}
-            alerts={
-              routesAndAlerts
-                ? routesAndAlerts[routeWithDirections.route.id]
-                : []
-            }
-          />
-        ))}
+        <div className="m-stop-page__departures--regular-screen mt-8">
+          {filteredRoutes.map(routeWithDirections => (
+            <RouteCard
+              key={routeWithDirections.route.id}
+              route={routeWithDirections.route}
+              directions={routeWithDirections.directions}
+              stop={stop}
+              alerts={
+                routesAndAlerts
+                  ? routesAndAlerts[routeWithDirections.route.id]
+                  : []
+              }
+            />
+          ))}
+        </div>
+        <div className="m-stop-page__departures--small-screen mt-8">
+          {/* This should only show on small screens */}
+          {shortFilteredRoutes.map(routeWithDirections => (
+            <RouteCard
+              key={routeWithDirections.route.id}
+              route={routeWithDirections.route}
+              directions={routeWithDirections.directions}
+              stop={stop}
+              alerts={
+                routesAndAlerts
+                  ? routesAndAlerts[routeWithDirections.route.id]
+                  : []
+              }
+            />
+          ))}
+          {!showMore && filteredRoutes.length > 2 && (
+            <div className="d-flex justify-content-center mt-10">
+              <button
+                className="btn btn-secondary"
```

Comment:
Add the `btn-block` class to this (the mockup has the button as full-width). Also if you move the `mt-10` class to here you won't need the wrapper `<div>`.